content/en/docs/tasks/administer-cluster/cluster-management.md

```diff
@@ -1,223 +0,0 @@
-
-Azure Kubernetes Service enables user-initiated resizing of the cluster from either the CLI or
-the Azure Portal and is described in the
-[Azure AKS documentation](https://docs.microsoft.com/en-us/azure/aks/scale-cluster).
-
-
-### Cluster autoscaling
-
-If you are using GCE or Google Kubernetes Engine, you can configure your cluster so that it is automatically rescaled based on
-pod needs.
-
-As described in [Compute Resource](/docs/concepts/configuration/manage-resources-containers/),
-users can reserve how much CPU and memory is allocated to pods.
-This information is used by the Kubernetes scheduler to find a place to run the pod. If there is
-no node that has enough free capacity (or doesn't match other pod requirements) then the pod has
-to wait until some pods are terminated or a new node is added.
-
-Cluster autoscaler looks for the pods that cannot be scheduled and checks if adding a new node, similar
-to the other in the cluster, would help. If yes, then it resizes the cluster to accommodate the waiting pods.
-
-Cluster autoscaler also scales down the cluster if it notices that one or more nodes are not needed anymore for
-an extended period of time (10min but it may change in the future).
-
-Cluster autoscaler is configured per instance group (GCE) or node pool (Google Kubernetes Engine).
-
-If you are using GCE then you can either enable it while creating a cluster with kube-up.sh script.
-To configure cluster autoscaler you have to set three environment variables:
-
-* `KUBE_ENABLE_CLUSTER_AUTOSCALER` - it enables cluster autoscaler if set to true.
-* `KUBE_AUTOSCALER_MIN_NODES` - minimum number of nodes in the cluster.
-* `KUBE_AUTOSCALER_MAX_NODES` - maximum number of nodes in the cluster.
-
-Example:
-
-```shell
-KUBE_ENABLE_CLUSTER_AUTOSCALER=true KUBE_AUTOSCALER_MIN_NODES=3 KUBE_AUTOSCALER_MAX_NODES=10 NUM_NODES=5 ./cluster/kube-up.sh
-```
-
-On Google Kubernetes Engine you configure cluster autoscaler either on cluster creation or update or when creating a particular node pool
-(which you want to be autoscaled) by passing flags `--enable-autoscaling` `--min-nodes` and `--max-nodes`
-to the corresponding `gcloud` commands.
-
-Examples:
-
-```shell
-gcloud container clusters create mytestcluster --zone=us-central1-b --enable-autoscaling --min-nodes=3 --max-nodes=10 --num-nodes=5
-```
-
-```shell
-gcloud container clusters update mytestcluster --enable-autoscaling --min-nodes=1 --max-nodes=15
-```
-
-**Cluster autoscaler expects that nodes have not been manually modified (e.g. by adding labels via kubectl) as those properties would not be propagated to the new nodes within the same instance group.**
-
-For more details about how the cluster autoscaler decides whether, when and how
-to scale a cluster, please refer to the [FAQ](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md)
-documentation from the autoscaler project.
-
-## Maintenance on a Node
```

Comment:
I am strongly against removing this subsection. It is not vendor specific content.

Reply:
I'm keen to leave instructions available. What I've (just) done is reword and retitle https:/k8s.io/docs/tasks/administer-cluster/safely-drain-node/ [[preview](https://deploy-preview-24415--kubernetes-io-master-staging.netlify.app/docs/tasks/administer-cluster/safely-drain-node/)].